specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -6003,6 +6204,395 @@
+          "readOnly": true
+        },
+        "status": {
+          "description": "Status of the job.",
+          "$ref": "#/definitions/JobStatus",
+          "readOnly": true
+        },
+        "progressMetrics": {
+          "description": "Progress metrics of the job.",
+          "$ref": "#/definitions/ProgressMetrics",
+          "readOnly": true
+        },
+        "statusMessages": {
+          "description": "Status messages of the job.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StatusMessage"
+          },
+          "readOnly": true
+        },
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "Created time of the job in UTC timezone.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "description": "Name and identifier for a job.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "description": "The job property dictionary. Properties can be added, but not removed or altered.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "tags": {
+          "description": "The job tag dictionary. Tags can be added, removed, and updated.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "LabelingJobResourceRequest": {
+      "description": "The parameters for creating or updating a labeling job.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/LabelingJob"
+        }
+      }
+    },
+    "LabelingJobResource": {
+      "description": "Machine Learning labeling job object wrapped into ARM resource envelope.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The resource URL of the entity (not URL encoded).",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the resource entity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The resource provider and type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/LabelingJob"
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        }
+      }
+    },
+    "LabelingJobResourcePaginatedResult": {
+      "type": "object",
+      "description": "Paginated list of Machine Learning labeling job objects wrapped in ARM resource envelope.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LabelingJobResource"
+          }
+        },
+        "continuationToken": {
+          "type": "string"
+        },
```

Comment:
Where is this used?

Reply:
This is used in the `$skipToken` [parameter](https://github.com/yetamsft/azure-rest-api-specs/blob/44754afdd63df701bd699c49cc74017d9dd62b75/specification/machineleaarningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json#L1949)of `labelingJobs_List` operation.

Reply:
I see. Since the $skipToken parameter is the input, I suppose this continuationToken here should be an output? If so maybe this should also be readonly

Reply:
Yes, this is marked as readonly.